resources/assets/coffee/osu_common.coffee

```diff
@@ -116,6 +116,9 @@ window.osu =
     $alert = $('.popup-clone').clone()
 
     closeAlert = -> $alert.click()
+    
+    # https://getbootstrap.com/docs/3.4/javascript/#alerts
+    disposeAlertWithFadeOut = -> $alert.fadeOut().alert('close')
```

Comment:
fade out being ignored aside, there's already logic for removing the element when clicked (the original callback) except it's not quite working right and that's the one that needs fixing (in `shared.coffee`)

The leaking problem also applies to warning/danger popups when clicking anywhere but close button for same reason.